File: Basic Problem Solving/SumofN-Numbers.cpp
#include<bits/stdc++.h>
using namespace std;


int main(){
    int n;
     cin >> n;
    int sum, count;
    sum = 0;
    count = 1 ;
    while (count <= n){
        sum += count;
        count = count + 1;

    }
    cout << sum << endl;
}

File: Basic Problem Solving/SumofN-Numbers.py
#using while loop
n = int(input())
count = 1
sum = 0
while (count <= n):
  sum += count
  count += 1
print(sum)

#using for loop
n = int(input())
sum = 0
for x in range(0, n + 1):
  sum += x

print(sum)
